playwright/e2e/crypto/backups-mas.spec.ts

```diff
@@ -24,8 +25,11 @@ test.describe("Encryption state after registration", () => {
         await page.getByRole("button", { name: "Continue" }).click();
         await registerAccountMas(page, mailpitClient, `alice_${testInfo.testId}`, "alice@email.com", "Pa$sW0rD!");
 
-        await app.settings.openUserSettings("Security & Privacy");
-        await expect(page.getByText("This session is backing up your keys.")).toBeVisible();
+        // Wait for the ui to load
+        await expect(page.locator(".mx_MatrixChat")).toBeVisible();
+
+        // xxx: why the backup key is not in 4S?
```

Comment:
There is no 4S at this point: recovery hasn't been set up.

Reply:
Oh, okay, I'll remove my comment then